lib/client.js

```diff
@@ -333,8 +333,7 @@ extend(Raven.prototype, {
       cb = kwargs;
       kwargs = {};
     } else {
-      // Do not use reference, as we'll modify this object later on
-      kwargs = kwargs ? JSON.parse(stringify(kwargs)) : {};
+      kwargs = utils.isPlainObject(kwargs) ? extend({}, kwargs) : {};
```

Comment:
`extend({}, 'something')` will treat each letter as a key so we want to avoid it